RecoLocalTracker/SiStripClusterizer/python/SiStripClusterizer_RealData_cfi.py

```diff
@@ -11,6 +11,12 @@
     cms.InputTag('siStripZeroSuppression','ScopeMode')),
                                )
 
+from Configuration.ProcessModifiers.approximatedClusters_cff import approximatedClusters
+from RecoLocalTracker.SiStripClusterizer.SiStripApprox2Clusters_cfi import SiStripApprox2Clusters
+SiStripApprox2Clusters.inputApproxClusters = 'SiStripClusters2ApproxClusters'
+SiStripApprox2Clusters.Clusterizer = cms.PSet(ConditionsLabel = cms.string(''))
```

Comment:
needed?

Reply:
yeah it's needed. otherwise you get the following python error, in Calibration/TkAlCaRecoProducers/python/ALCARECOSiStripCalZeroBias_cff.py, at the line calZeroBiasClusters.Clusterizer.ConditionsLabel = 'unbiased':
TypeError: ConditionsLabel does not already exist, so it can only be set to a CMS python configuration type

Reply:
No, because you can edit that file and configure the clone of the clusterizer not not need that useless parameter.